xray/commands/audit/scan.go

```diff
@@ -26,14 +26,20 @@ const (
 
 type FileContext func(string) parallel.TaskFunc
 type indexFileHandlerFunc func(file string)
+type PrintFormat string
+
+const (
+	Table = "table"
+	Json  = "json"
```

Comment:
Let's give the values the exact type. Also, there is already a `const` block in line 23. I'd prefer to merge them into 1 block.
```suggestion
	Table PrintFormat = "table"
	Json  PrintFormat = "json"
```